### ISSUE TITLE
Cross Entropy Loss Function

### ISSUE THREAD

**Issue by izzat5233** [izzat5233/fruit-classifier-wasm#8]
Cross-entropy is a measure of the difference between two probability distributions for a given random variable or set of events.
Its an important loss function that should be added and implemented in `loss` namespace.
